packages/web-components/src/components/va-back-to-top/va-back-to-top.css

```diff
@@ -43,7 +44,6 @@ a {
   padding: 0;
   pointer-events: none;
   position: absolute;
-  transition: all 0.25s ease-in;
```

Comment:
I don't think this should be removed. We want the component to transition when it comes into view. You can see how it fades in on this page here: https://www.va.gov/burials-memorials/memorial-items/

Reply:
We could change the `all` to `opacity` so that it only transitions the opacity and not the focus outline.

Reply:
@Andrew565 That's a good idea. It might also need some kind of positioning transition too? It kind of moves up when it comes in view:


https://github.com/user-attachments/assets/578fb9b0-4a78-478d-8cfb-37c366b17a94

Reply:
I was able to fix it with background color transition, and the positioning transition works as well.